yt_dlp/extractor/tenplay.py

```diff
@@ -113,3 +113,62 @@ def _real_extract(self, url):
             'uploader': 'Channel 10',
             'uploader_id': '2199827728001',
         }
+
+
+class TenPlaySeasonIE(TenPlayIE):
```

Comment:
Do not subclass from concrete IE; move common code into a baseclass and subclass both concrete IEs from the baseclass

Reply:
Actually, it looks like none of the code from `TenPlayIE` is even needed, you can just subclass from `InfoExtractor` instead